data/json/skills.json

```diff
@@ -53,15 +46,15 @@
   {
     "type": "skill",
     "id": "driving",
-    "name": { "str": "driving" },
-    "description": "Your skill in operating and steering a vehicle in motion.  A higher level allows greater control over vehicles at higher speeds, and reduces the penalty of shooting while driving.",
+    "name": { "str": "vehicles" },
+    "description": "Your skill in operating and steering a vehicle in motion, from cars and boats to helicopters and hot air balloons.  A higher level allows greater control over vehicles at higher speeds, and reduces the penalty of shooting while driving.",
     "display_category": "display_interaction"
   },
   {
     "type": "skill",
     "id": "swimming",
-    "name": { "str": "swimming" },
-    "description": "Your ability to stay afloat and move around in bodies of water.  This skill keeps you from drowning, affects your combat effectiveness and speed in deep water, and determines the detriment of swimming with heavier gear.",
+    "name": { "str": "athletics" },
+    "description": "Your ability to swim, climb things, and other feats of physical skill.  Affects swimming endurance and ability to swim with weight.",
```

Comment:
I don't think it's a good idea to put yet-unimplemented things into descriptions. Athlitics does NOT currently help with climbing, the description is outright lying to players.

Reply:
Yeah, I have been debating that as well. I should probably change it to indicate that's the plan, because I want people to be aware that it might be worth investing in in the near term